entities/regions.py

```diff
@@ -0,0 +1,14 @@
+from random import randint
+from random import choice
+from config import Config
+
+class Regions(object):
+    region = []
+    def create_region(self):
+        for i in range(Config.number_of_regions):
+            self.region_id = i + 1
+            for j in range(Config.number_of_subregions):
+                self.hub = [int((j + 1 + i * 4) * Config.subregion_length - (Config.subregion_length / 2)),
```

Comment:
`hub =`. Don't use `self.hub` here.

Hub should preferably be an object, so make it into a class that we can extend upon.

Reply:
No need for `int()`. Do `// 2` instead of `/ 2`, as this forces [integer divison](https://en.wikipedia.org/wiki/Division_(mathematics)#Of_integers) (see item 5 there).